AxonDeepSeg/data_management/input_data.py

```diff
@@ -159,8 +160,8 @@ def __init__(self, trainingset_path, config, type_ ='train', batch_size = 8, pre
         self.epoch_size = len(self.samples_list)
         self.preload_all = preload_all
         self.loaded_data = None
-        self.mean = config['dataset_mean']
-        self.variance = config['dataset_variance']
+        #self.mean = config['dataset_mean']
```

Comment:
@alzaia Did you mean to commit these commented out lines? I ask because of the commented out print statement above – looks like you were debugging.

Reply:
Yes it was intended, we do not currently use the dataset mean and variance in that way (and it is not very practical because the user would need to compute it and use it as input param. in the config.). So for now we are using the `apply_legacy_preprocess` function of `patch_management_tools.py` instead of the `apply_preprocess` function (we automatically compute the mean and variance but for each patch).

Reply:
@alzaia Ok – thanks for clarifying!